docs/source/core.rst

```diff
@@ -447,9 +469,56 @@ The ``CurrencyValue`` class represents a value tied to a currency. The methods i
 - ``gte(other)`` - checks if this value is greater than or equal to the other value. The argument must be a CurrencyValue with the same Currency.
 - ``isZero()`` - returns true if the value is zero.
 
+.. _TransactionOptions:
+
+TransactionOptions
+==================
+
+Represents a options for sending transactions.
+All fields are optional.
+
+Fields:
+
+- ``signer?: Signer`` - specifies `signer <https://docs.ethers.io/v5/api/signer/#Signer>`_ for a transaction.
+
+- ``transactionName?: string`` - specifies a transaction name.
```

Comment:
Perhaps add something more about name e.g. "Used for transaction history nad notifications."